addons/portal/static/src/embed/frontend/chatter_root.js

```diff
@@ -0,0 +1,21 @@
+/* @odoo-module */
+
+import { Chatter } from "@mail/chatter/web_portal/chatter";
+
+import { Component, xml, useSubEnv } from "@odoo/owl";
+
+export class ChatterRoot extends Component {
+    static template = xml`
+        <Chatter threadId="props.resId" threadModel="props.resModel" hasComposer="props.hasComposer" twoColumns="props.twoColumns"/>
```

Comment:
Nitpicking: `hasComposer` could simply be renamed `composer="false/true"`.

I don't know what the `twoColumns` props implies. I feel this could be better worded and have a more meaningful value than a boolean, maybe `columns=1/2` _(not saying this is a correct alternative to choose, but just saying this prop is likely not idiomatic enough as it is)_

Out-of-scope of PR: probably should rename `threadModel` and `threadId` to `model` and `id`, to simplify props names